src/protocol/sender/buffers.rs

```diff
@@ -36,7 +36,7 @@ impl TransmitBuffer {
 
     /// In the case of a message longer than the packet size,
     /// It will be split into multiple packets
-    pub fn push_message(&mut self, data: (Instant, Bytes)) {
+    pub fn push_data(&mut self, data: (Instant, Bytes)) {
```

Comment:
I'd prefer this keeps the push_message name, it's a bit more descriptive I think.

Reply:
Ok, I'm not sure why I changed this. I agree.